hedera-node/docs/system-accounts-operations.md

```diff
@@ -0,0 +1,184 @@
+# System accounts
+
+## Definition
+The Hedera network reserves the first 1000 entity numbers for its own uses, and any account with a number in this reserved range is referred to as a system account.
+
+The following addresses are reserved for the Hedera precompiled contracts:
+- 0x167 (0.0.359) - HTS precompile;
+- 0x168 (0.0.360) - ExchangeRate precompile;
+- 0x169 (0.0.361) - PRNG precompile.
+
+Some of the system accounts exist today and more can be created in the future. Certain system accounts have predefined roles in the network. These include:
+- 0.0.2 - treasury account;
+- 0.0.50 - system admin;
+- 0.0.55 - address book admin;
+- 0.0.57 - exchange rates admin;
+- 0.0.58 - freeze admin;
+- 0.0.59 - system delete admin;
+- 0.0.60 - system undelete admin;
+- 0.0.800 - staking reward account;
+- 0.0.801 - node reward account.
+
+Addresses in range 1 → 9 are special addresses on which the Ethereum precompiled contracts exist. This is valid for both Ethereum and Hedera.
+
+Lastly, the system accounts can be divided in these two groups:
+- addresses in range 0 → 750 - accounts that reject hbar transfers. An exception is the HTS precompiled contract that can receive transfers.
```

Comment:
Also why is 0x167 (0.0.359) able to receive HBAR?! 
```suggestion
- addresses in range 0 → 750 - accounts that reject hbar transfers. An exception is the HTS system contract that can receive transfers.
```

Reply:
Only for token create cases. I will clarify that in the doc.

Reply:
Just for the token create scenario. This was explained in the last section regarding the transfers but I will clarify that here as well.